operations/patch.go

```diff
@@ -144,6 +145,20 @@ func Patch() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
+			params.setDefaultProject(conf)
+
+			for _, project := range conf.Projects {
+				if errStrs := model.ValidateProjectAliases(project.LocalAliases, "Local Aliases"); errStrs != nil {
```

Comment:
I don't think when a user wants to create a patch we _always_ need to validate all aliases in their project. We should be able to instead do this in setLocalAliases just for any aliases that are used I'd think.